ansible/vars.yml

```diff
@@ -9,20 +9,20 @@ postgres_major:
 
 # Full version strings for each major version
 postgres_release:
-  postgresorioledb-17: "17.0.1.56-orioledb"
-  postgres17: "17.4.1.006"
-  postgres15: "15.8.1.063"
+  postgresorioledb-17: "17.0.1.57-orioledb"
+  postgres17: "17.4.1.007"
+  postgres15: "15.8.1.064"
```

Comment:
> sorry also please advance the versions in ansible/vars.yml one more than where they are at in develop branch please

These are the versions to advance, right?

Reply:
@laurenceisla yeah you got it. If you need to test in local infra prior to merge to develop you can also add a suffix to those as well, and then create a pre release ami to test in local infra too.

Reply:
@samrose I'm having some issues pinpointing why the tests are not running. The AMI builds pass correctly, but it times out waiting for the postgrest service when running the tests.

https://github.com/supabase/postgres/actions/runs/14246128670/job/39927529001#step:13:275

I tried testing on a local environment and I managed to run the infra stack, but I'm not sure how to generate a Postgres pre-release to add to the middleware and test this issue locally:

> you can also add a suffix to those as well, and then create a pre release ami to test in local infra too.

Are there any docs I can follow?

Reply:
I sent Laurence some info on generating an AMI